server/base/application.ts

```diff
@@ -25,7 +25,7 @@ export class Application {
   private static app: express.Application = express();
 
   public static async START(): Promise<void> {
-    this.app.use(express.static(path.join(__dirname, '../client')));
+    this.app.use(express.static(path.join(__dirname, '../../client')));
```

Comment:
Shouldn't be `../../out/client`?

Reply:
`application.ts` will be build into `out/server/application.js`
So static path should be set by considering runtime path.